Stripe/STPEphemeralKeyManager.m

```diff
@@ -64,18 +68,44 @@ - (void)handleWillForegroundNotification {
     // eager refreshses to once per hour.
     if (!self.currentKeyIsUnexpired && self.shouldPerformEagerRefresh) {
         self.lastEagerKeyRefresh = [NSDate date];
-        [self.keyProvider createCustomerKeyWithAPIVersion:self.apiVersion completion:^(NSDictionary *jsonResponse, __unused NSError *error) {
-            STPEphemeralKey *key = [STPEphemeralKey decodedObjectFromAPIResponse:jsonResponse];
-            if (key) {
-                self.customerKey = key;
-            }
+        [self getOrCreateKey:^(__unused STPEphemeralKey * _Nullable ephemeralKey, __unused NSError * _Nullable error) {
+            
```

Comment:
It looks like `self.ephemeralKey` gets set in the actual to `getOrCreateKey`. Could we add some comments here explaining why this is called and why we do nothing with the response here?

Reply:
👍